CHANGES.txt

```diff
@@ -21,6 +21,10 @@ Changes
 - Greatly improved performance of queries that uses scalar functions inside the
   ``WHERE`` clause.
 
+- In certain cases it's no longer necessary to explicitly add generated columns
```

Comment:
shouldn't we always put new stuff on top of the list here?

Reply:
We've no requirement that the changes are listed in a chronological order, but you're right, we usually add new entries to the top